angular.json

```diff
@@ -41,6 +41,33 @@
             ]
           },
           "configurations": {
+            "staging": {
```

Comment:
@pegnott It appears the staging configuration is defined twice. I would remove this from your PR. Otherwise you should be gtg

Reply:
updated to remove duplicate staging ref